conf/etc/v2/ha.conf.template

```diff
@@ -22,7 +22,7 @@ EVENT_MANAGER:
     message_type: health_events
     producer_id: system_health
     consumer_group: health_monitor
-    consumer_id: health_monitor
+    consumer_id: 1
```

Comment:
Please update this in LLD if not done already.

Reply:
Updated